biodatasets/medhop/medhop.py

```diff
@@ -0,0 +1,241 @@
+abstracts from PubMed, and the queries are about interactions between
+pairs of drugs. The correct answer has to be inferred by combining
+information from a chain of reactions of drugs and proteins.
+"""
+
+_DATASETNAME = "MedHop"
+
+_HOMEPAGE = "http://qangaroo.cs.ucl.ac.uk/"
+
+_LICENSE = "CC BY-SA 3.0"
+
+_BASE_GDRIVE = "https://drive.google.com/uc?export=download&confirm=yTib&id="
+
+_URLs = {
+    "source": _BASE_GDRIVE + "1ytVZ4AhubFDOEL7o7XrIRIyhU8g9wvKA",
+    "bigbio_qa": _BASE_GDRIVE + "1ytVZ4AhubFDOEL7o7XrIRIyhU8g9wvKA",
+}
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+
+class MedHopDataset(datasets.GeneratorBasedBuilder):
+    """MedHop"""
+
+    DEFAULT_CONFIG_NAME = "medhop_source"
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="medhop_source",
+            version=SOURCE_VERSION,
+            description="MedHop source schema",
+            schema="source",
+            subset_id="MedHop",
+        ),
+        BigBioConfig(
+            name="medhop_bigbio_qa",
+            version=BIGBIO_VERSION,
+            description="MedHop BigBio schema",
+            schema="bigbio_qa",
+            subset_id="MedHop",
+        ),
+    ]
+
+    def _info(self):
+
+        if self.config.schema == "source":
+
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "type": datasets.Value("string"),
+                    "body": datasets.Value("string"),
+                    "documents": datasets.Sequence(datasets.Value("string")),
+                    "concepts": datasets.Sequence(datasets.Value("string")),
+                    "ideal_answer": datasets.Sequence(datasets.Value("string")),
+                    "exact_answer": datasets.Sequence(datasets.Value("string")),
+                    "triples": [
+                        {
+                            "p": datasets.Value("string"),
+                            "s": datasets.Value("string"),
+                            "o": datasets.Value("string"),
+                        }
+                    ],
+                    "snippets": [
+                        {
+                            "offsetInBeginSection": datasets.Value("int32"),
+                            "offsetInEndSection": datasets.Value("int32"),
+                            "text": datasets.Value("string"),
+                            "beginSection": datasets.Value("string"),
+                            "endSection": datasets.Value("string"),
+                            "document": datasets.Value("string"),
+                        }
+                    ],
+                }
+            )
+
+        # simplified schema for QA tasks
+        elif self.config.schema == "bigbio_qa":
+
+            features = datasets.Features(
```

Comment:
we should be importing the features here (also will need to add `from utils import schema` at the top). I noticed this was not updated in the bioasq.py example too and updated it there. 

```suggestion
            features = schemas.qa_features
```

Reply:
You mean `from utils import schemas` not `from utils import schema` right ?

Reply:
yes, good catch ... I'll update the comment in case anyone ends up here looking for clues.